test/asm/stop-section.asm

```diff
@@ -0,0 +1,4 @@
+SECTION "test", ROM0
+	db 1
+SECTION STOP
```

Comment:
This should be `ENDS` for consistency with other constructs

Reply:
I didn't like the original proposal of `ENDS` because (a) "ends" is an English word that may well be a user-defined symbol already; and (b) if we really do want it as a keyword, it would be more frequent+valuable later to end a struct. How about `ENDSECTION`?

Reply:
Yeah, sure.